app/router.js

```diff
@@ -20,7 +20,12 @@ Router.map(function () {
     this.route('login');
     this.route('logout');
     if (ENV.controllerLogin !== 'true') {
-      this.route('switch');
+      if (ENV.acmidm.clientId === '{{OAUTH_API_KEY}}') {
+        this.route('mock-login');
+      }
+      this.route('switch-login');
+    } else {
+      this.route('controller-login');
```

Comment:
**Question:** Is it normal that login routes are defined twice, in `/authorization` and `/` level?

https://github.com/lblod/frontend-contactgegevens-loket/blob/98042a4f93333547650fa1eabf716b48fdb6b539/app/router.js#L9-L14

Reply:
yes, we defined login for the index page and a login for authorization of acm/idm